src/methods/defining_methods/hard/q5/FavMovie.java

```diff
@@ -0,0 +1,39 @@
+package methods.defining_methods.hard.q5;
+
+/**
+ * [J] Write a Java Program that asks users for their favourite movie and asks them how many times they've watched that movie. Then,
```

Comment:
Change question text to:
`Write a Java program asking users for their favourite movie and how many times they've watched that movie. Then, write a Java Method that takes the string and integer as parameters and returns a string. The returned string should be as follows: if the integer is less than or equal to five, then return a string saying: "You've watched <the movie name> a few times!"; if the integer is less than or equal to 10 and greater than 5, return a string saying: "You've watched <the movie name> a lot of times!"; if the integer is greater than 10, then return a string saying: "You've watched <the movie name> so many times! Wow!". Call your method movieCounter and pass in the movie name and the number of times you have watched the movie respectively.`